apps/marketplace/components/Brief/BriefDownloadResponses.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import AUheading from '@gov.au/headings/lib/js/react.js'
+import AUbutton from '@gov.au/buttons/lib/js/react.js'
+import AUpageAlert from '@gov.au/page-alerts/lib/js/react.js'
+import { rootPath } from 'marketplace/routes'
+import styles from './BriefDownloadResponses.scss'
+
+export class BriefDownloadResponses extends Component {
+  constructor(props) {
+    super(props)
+
+    this.handleButtonClick = this.handleButtonClick.bind(this)
+    this.getFileSizeAndType = this.getFileSizeAndType.bind(this)
+  }
+
+  getFileSizeAndType() {
+    const bytes = this.props.brief.responsesZipFilesize
+    let result = ''
+    if (this.props.brief.lot === 'digital-professionals') {
+      let size = ''
+      if (bytes < 1048576) {
```

Comment:
Could this go in some kind of utils file?  Seems like something that could be useful on other pages?